pkgs/applications/networking/remote/anydesk/default.nix

```diff
@@ -5,7 +5,7 @@
 
 let
   sha256 = {
-    x86_64-linux = "19751ygq1ng79aniqx91qawc0cw07cwdjdjd88azc9ww6z6nv0mp";
+    x86_64-linux = "1qbq6r0yanjappsi8yglw8r54bwf32bjb2i63awmr6pk5kmhhy3r";
     i386-linux   = "0dwc7v4p1dz51444zwn0kds23yi87r4h2d3isfj9xwkn90pxb7in";
```

Comment:
6.1.0 has dropped support for 32Bit. Could you remove it here too?

Reply:
Done.